### PR TITLE
Improving English translation

### PR DESCRIPTION
**Pull request by lirielm**

### CODE REVIEW

File: en.html [hunk 1]
```diff
@@ -29,10 +29,10 @@
         <p class="title">about</p>
         <div class="about">
           <p>
-            <strong>ruby</strong> is a dynamic, open source programming language with a focus on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
+            <strong>Ruby</strong> is a dynamic, open-source programming language focused on simplicity and productivity. It has an elegant syntax that is natural to read and easy to write.
```
Comment (hugonomura) [inline en.html]: please, keep the text in lower case. this is the pattern in the text. :)

File: en.html [hunk 2]
```diff
@@ -29,10 +29,10 @@
...
           </p>
           <p>
-            <strong>ruby on rails</strong> is a web development framework(free and open source), optimized to increase the programmers' productivity and his delight with coding. It allows you to write ellegant code, favoring standards over customization.
+            <strong>Ruby on Rails</strong> is a web development framework(free and open-source), optimized to increase the programmers' productivity and enjoyment while coding. It allows you to write ellegant code, favoring standards over customization.
```
Comment (hugonomura) [inline en.html]: please, keep the text in lower case. this is the pattern in the text. :)

**Review by hugonomura — CHANGES_REQUESTED** [review]

File: en.html (hunk 1 above)
Comment (lirielm) [reply en.html]: I'll change it, but, it is not grammatically correct then :)

File: en.html (hunk 2 above)
Comment (lirielm) [reply en.html]: I'll change it, but, it is not grammatically correct then :)

File: en.html (hunk 1 above)
Comment (hugonomura) [reply en.html]: Yeah, it is strange, but the pt-br page follow this pattern too.

**Comment by hugonomura** [comment]
hey @lucasrenan take a look in this PR

File: en.html (hunk 1 above)
Comment (lucasrenan) [reply en.html]: it's more about typographic than grammar :D

**Comment by lucasrenan** [comment]
@lirielm thanks for your contribution!

@hugonomura thanks for review it.

**Pull request merged**